src/cromshell/list_outputs/command.py

```diff
@@ -0,0 +1,200 @@
+        api_workflow_id=config.cromwell_api_workflow_id,
+        timeout=config.requests_connect_timeout,
+        verify_certs=config.requests_verify_certs,
+        headers=http_utils.generate_headers(config),
+    )
+
+    return filer_outputs_from_workflow_metadata(workflow_metadata)
+
+
+def filer_outputs_from_workflow_metadata(workflow_metadata: dict) -> dict:
+    """Get the outputs from the workflow metadata
+
+    Args:
+        workflow_metadata (dict): The workflow metadata
+    """
+    calls_metadata = workflow_metadata["calls"]
+    output_metadata = {}
+    extract_task_key = "outputs"
+
+    for call, index_list in calls_metadata.items():
+        if "subWorkflowMetadata" in calls_metadata[call][0]:
+            output_metadata[call] = []
+            for scatter in calls_metadata[call]:
+                output_metadata[call].append(
+                    filer_outputs_from_workflow_metadata(scatter["subWorkflowMetadata"])
+                )
+        else:
+            output_metadata[call] = []
+            for index in index_list:
+                output_metadata[call].append(index.get(extract_task_key))
+
+    return output_metadata
+
+
+def print_task_level_outputs(output_metadata: dict) -> None:
+    """Print the outputs from the workflow metadata
+    output_metadata: {call_name:[index1{output_name: outputvalue}, index2{...}, ...], call_name:[], ...}
+
+    Args:
+        output_metadata (dict): The output metadata from the workflow
+    """
+    for call, index_list in output_metadata.items():
+        print(call)
+        for call_index in index_list:
+            if call_index is not None:
+                print_file_like_value_in_dict(outputs_metadata=call_index, indent=True)
+
+
+def print_file_like_value_in_dict(outputs_metadata: dict, indent: bool) -> None:
+    """Print the file like values in the output metadata dictionary
+
+    Args:
+        outputs_metadata (dict): The output metadata
+        indent (bool): Whether to indent the output
+    """
+
+    for output_name, output_value in outputs_metadata.items():
+        if isinstance(output_value, str):
+            print_output_name_and_file(output_name, output_value, indent=indent)
+        elif isinstance(output_value, list):
+            for output_value_item in output_value:
+                print_output_name_and_file(
+                    output_name, output_value_item, indent=indent
+                )
+
+
+def print_output_name_and_file(
+    output_name: str, output_value: str, indent: bool = True
+) -> None:
+    """Print the task name and the file name
+
+    Args:
+        output_name (str): The task output name
+        output_value (str): The task output value
+        indent (bool): Whether to indent the output"""
+
+    i = "\t" if indent else ""
+
+    if isinstance(output_value, str):
+        if is_path_or_url_like(output_value):
+            print(f"{i}{output_name}: {output_value}")
+
+
+def is_path_or_url_like(in_string: str) -> bool:
```

Comment:
I'm thinking about future-proofing us a little here: how about we classify the `in_string`? 
I mean classify it as a local regular path, GCS path, or Azure path, or not a path at all.